lua/nvim-surround/config.lua

```diff
@@ -367,8 +367,15 @@ M.get_delimiters = function(char, line_mode)
     end)()
```

Comment:
It seems that there's quite a bit of redundant code in `get_delimiters` that I forgot to get rid of; do you mind trying the following for me and seeing if it still passes test cases?

* Remove the `if not char` check (the other functions should all properly propagate `nil`)
* Replace the immediately invoked lambda on line 361 with `local delimiters = M.get_add(char)(char)`, as `get_add` should handle the invalid case already

Reply:
It passes all tests on my machine before and after those changes.